### ISSUE TITLE
Preloader for processing

### ISSUE THREAD

**Issue by diosby** [rangav/thunder-client-support#464]
Dear @rangav.

**Is your feature request related to a problem? Please describe.**
I have some requests that take a lot of time and I can do several requests per at the same time. Unfortunately, I cannot see when the requests have been done and I check them each time through opening the requests.  

**Describe the solution you'd like**
I would like to see preloader for requests in the list of requests. 

**Describe alternatives you've considered**
If the preloader isn't possible a message about processing also can be here.

**Implementation:**
![Screenshot from 2022-02-09 11-52-26](https://user-images.githubusercontent.com/19969187/153161741-66e2e253-f811-49bf-9295-1585e3ca290f.png)

Thank you for your previous improvements!

**Comment by rangav** [comment]
Thanks @diosby for the feedback, 

Let me check if got it right 

- So you want send a request
- Because it's taking long time, you want to close the window
- And see a processing status in the sidebar
- When it's completed then you want check the response right?

**Comment by diosby** [comment]
@rangav Yes, I switch on another request, but the last one is loading and I am waiting it. 
If it is possible I would like to see the status at least two places:
- In an item of the request.
- If the Client isn't opened (or always or when a number is more then one) then shows a number of active requests or preloader or something else on the icon of the client in the side menu.

Examples: 
![Screenshot from 2022-02-10 14-30-32](https://user-images.githubusercontent.com/19969187/153400650-5a3482f3-6f20-4ddb-8a80-9747ac498801.png)
![Screenshot from 2022-02-10 14-33-00](https://user-images.githubusercontent.com/19969187/153401242-9e4e5981-f31d-4c7c-b111-6b3479cb331d.png)

**Comment by rangav** [comment]
@diosby got it, will review and add to roadmap.
